src/Connection/AsyncTcpConnection.php

```diff
@@ -243,7 +243,7 @@ public function connect(): void
         }
 
         if (!$this->eventLoop) {
```

Comment:
Use Null coalescing assignment operator
From PHP/7.4

`$this->eventLoop ??= Worker:getEventLoop();`